2-structured-data/books/model-cloudsql.js

```diff
@@ -137,8 +137,8 @@ function createSchema (config) {
 
   connection.query(
     `CREATE DATABASE IF NOT EXISTS \`bookshelf\`
-      DEFAULT CHARACTER SET = \'utf8\'
-      DEFAULT COLLATE \'utf8_general_ci\';
+      DEFAULT CHARACTER SET = 'utf8'
```

Comment:
I typically like to avoid unrelated changes, but I'll allow it.

Just to make sure, does this not break anything in SQL?